app/models/order_detail.rb

```diff
@@ -220,7 +219,7 @@ def self.with_estimated_costs
 
   def in_review?
     # check in the database if self.id is in the scope
-    self.class.all_in_review.find_by_id(id) ? true : false
+    self.class.all_in_review.find_by(id: id) ? true : false
```

Comment:
Could this use `self.class.all_in_review.exists?(id: id)` to get rid of the ternary?